aspnetcore/blazor/fundamentals/configuration.md

```diff
@@ -126,7 +126,7 @@ var client = new HttpClient()
     BaseAddress = new Uri(builder.HostEnvironment.BaseAddress)
 };
 
-builder.Services.AddTransient(sp => client);
+builder.Services.AddScoped(sp => client);
```

Comment:
This one still seems strange to me.

I'll bet I'm just missing something, and I understand that scopes are a little different in Blazor WASM, but the `client` instance is essentially a singleton in this case, yet is being exposed as though it's being created per-scope. In the case where the owning scope was disposed, `client` would also be disposed and continue to be given back for other scopes.

I know it _works_,  but I just wanted to mention this in passing.